Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN pip install virtualenv
 
 #install ffmpeg
 RUN cd /tmp \
-  && wget -O ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-i686-static.tar.xz \
+  && wget -O ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-armel-static.tar.xz \
```

Comment:
This will depend on the architecture of the host. My guess is that you're using a raspberry. But changing that will make the x86 architecture fail...

We have to work on a multi architecture build.

Reply:
yep. we need to open this an an issue. I've started to get a multi-arch build system in place, and this will need to be one of the things we need to hit for that milestone.

Reply:
That definitely makes sense. I wrongly assumed that this was for raspberry.

Reply:
I just created the first step of a multi-stage multi-arch build in #19 , and once accepted, the following PR will result in a `docker pull octoprint/octoprint` being "architecture aware" and should have a compatible ffmpeg.

For now, this image hasn't hit it's true "1.0" yet, so the ffmpeg functionality (as well as cura) is pretty much suspect anyways.